pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -54,6 +54,14 @@ type DefaultPreemption struct {
 	podLister corelisters.PodLister
 	pdbLister policylisters.PodDisruptionBudgetLister
 	Evaluator *preemption.Evaluator
+
+	// EligiblePods returns victim pods which are allowed to be preempted by the provided preemptor.
+	// The default behavior is to allow any pods of lower priority to be preempted by any pods of higher priority.
+	EligiblePods func(nodeInfo *framework.NodeInfo, preemptor *v1.Pod) []*framework.PodInfo
+
+	// OrderedPods sorts eligible victims in-place in descending order of highest to lowest priority.
+	// Pods at the start of the slice are less likely to be preempted.
+	OrderedPods func(eligible []*framework.PodInfo)
```

Comment:
Will `OrderPods` be better since this returns a function instead of a sorted pod slice

Reply:
Yes that sounds better, I wasn't super sure about the naming myself. Renamed here and throughout